core/build.gradle

```diff
@@ -35,7 +35,7 @@ dependencies {
 
     implementation 'androidx.core:core-ktx:1.9.0'
     implementation 'androidx.lifecycle:lifecycle-runtime-ktx:2.5.1'
-    implementation 'androidx.activity:activity-compose:1.7.0-beta01'
+    implementation 'androidx.activity:activity-compose:1.6.1'
```

Comment:
Beta no bueno?

Reply:
It was a leftover from when I was cleaning up, didn't see a point in it being in its own pr as a small thing